db/migrate/20150621061920_create_daily_progresses.rb

```diff
@@ -0,0 +1,12 @@
+class CreateDailyProgresses < ActiveRecord::Migration
+  def change
```

Comment:
__[reek]__
* [TooManyStatements] has approx 6 statements
* [FeatureEnvy] refers to t more than self